src/pages/ProfileScreen.js

```diff
@@ -56,22 +74,49 @@ export class ProfileScreen extends React.Component {
 class Info extends React.Component {
     constructor(props) {
         super(props);
-        this.state = {
-            text : ''
+        switch (this.props.name) {
+            case 'Name':
+                this.state = {text:'Nguyen Van A'}
+                break;
+            case 'Email':
+                this.state = {text:'iamA@gmail.com'}
+                break;
+            case 'Phone':
+                this.state = {text:'0123456789'}
+                break;
+            default:
+                break;
         }
     }
     render() {
+        
         return (
             <View style = {styles.infocontainer}>
                 <Text style ={{fontSize : 15, flex : 1}}> {this.props.name} </Text>
-                <View style = {{flex : 4, padding : 5, borderColor : '#000000', borderWidth : 1}}>
+                {/* <View style = {{flex : 4}}>
```

Comment:
Keeping the code clean by getting rid of commented-out code.
Feel free to delete unused code. Don't worry about it because you can see the history on GitHub.